COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,12 @@
+# Table of Contents
+- [Releasing](#releasing)
+
+# Releasing
+
+- ensure the tests are passing on master
+- version on npm `npm version [patch|minor|major]`
+- push changes from versioning to master, and the new tags created `git push && git push --tags`
```

Comment:
We may want to specify that the repo should be clean or to push only the new tag, or else someone may inadvertently push their own private tags up.